webhooks-extension/README.md

```diff
@@ -1,18 +1,72 @@
+webhooks-extension/README.md
+
 # Webhooks Extension
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/kubernetes/experimental/blob/master/LICENSE)
 
-The Webhooks Extension for Tekton allows users to set up Git webhooks that will trigger Tekton PipelineRuns and TaskRuns. An initial implementation will use Knative Eventing but we're closely following the eventing discussion in  [Tekton Pipeline](https://github.com/tektoncd/pipeline) to minimize necessary componentry.
+The Webhooks Extension for Tekton allows users to set up GitHub webhooks that will trigger Tekton `PipelineRuns` and associated `TaskRuns`. This initial implementation utilises Knative Eventing but we're closely following the eventing discussion in  [Tekton Pipeline](https://github.com/tektoncd/pipeline) to minimize necessary componentry.
 
 In addition to Tekton/Knative Eventing glue, it includes an extension to the Tekton Dashboard.
 
-## Runtime Dependencies
-- [Tekton](https://github.com/tektoncd/pipeline) & [Tekton Dashboard](https://github.com/tektoncd/dashboard)
-- Knative [eventing](https://knative.dev/docs/eventing/), [eventing sources](https://knative.dev/docs/eventing/sources/), & [serving](https://knative.dev/docs/serving/)
+## Prerequisites
+- Install [Tekton](https://github.com/tektoncd/pipeline/blob/master/docs/install.md) _(version 0.3.0 is recommended)_ and the [Tekton Dashboard](https://github.com/tektoncd/dashboard)
+
+- Install Istio - for a quickstart install, for example of version 1.1.8 run `./scripts/install_istio.sh 1.1.8` _or_ follow https://knative.dev/docs/install/installing-istio/ for a more customised install _(Istio version 1.1.8 is recommended)_
+
+- Install Knative Eventing, Eventing Sources & Serving - for a quickstart install, for example of version 0.6.0 run `./scripts/install_knative.sh v0.6.0`, for more detailed instructions see the [Knative docs](https://knative.dev/docs/install/index.html) _(Knative version 0.6.0 is recommended)_
+
+*Running on Docker for Desktop?*
+
+- You must have the `edge` version enabled and supply your local cluster with sufficient resources _(6 CPUs, 10GiB Memory & 2.5GiB swap is known good)_
+
+## Install Quickstart
+
+Until published images are available ([Issue 28](https://github.com/tektoncd/experimental/issues/28)), the webhooks extension must be built and deployed manually following the below steps:
+
+### Using docker only - kubectl apply
+
+1. Build and push the extension and sink images to your own image registry e.g. Dockerhub
+
+    `./scripts/build_extension.sh [your_image_registry]`
```

Comment:
would [my_image_registry] be 'mnuttall', 'mnuttall/extension' or 'extension' ? And how do I build the sink image?

Reply:
Looking at the code I think it would be 'mnuttall' for docker hub